landbosse/model/CollectionCost.py

```diff
@@ -694,8 +788,23 @@ def calculate_costs(self, calculate_costs_input_dict, calculate_costs_output_dic
         calculate_costs_output_dict['Equipment Cost USD without weather delays'] = (calculate_costs_output_dict['Days taken for trenching (equipment)'] * calculate_costs_output_dict['Equipment cost of trenching per day {usd/day)'])
         calculate_costs_output_dict['Equipment Cost USD with weather delays'] = calculate_costs_output_dict['Equipment Cost USD without weather delays'] *  calculate_costs_output_dict['wind_multiplier']
 
-        trenching_equipment_rental_cost_df = pd.DataFrame([['Equipment rental',calculate_costs_output_dict['Equipment Cost USD with weather delays'], 'Collection']],
-                                               columns = ['Type of cost', 'Cost USD', 'Phase of construction'])
+        if calculate_costs_input_dict['turbine_rating_MW'] >= 0.1:
+            trenching_equipment_rental_cost_df = pd.DataFrame([['Equipment rental', calculate_costs_output_dict[
+                'Equipment Cost USD with weather delays'], 'Collection']],
+                                                              columns=['Type of cost', 'Cost USD',
+                                                                       'Phase of construction'])
+        else:
+            if calculate_costs_output_dict['Equipment Cost USD with weather delays'] < 137:
+                calculate_costs_output_dict['Equipment Cost USD with weather delays'] = 137   #cost of renting for a day
```

Comment:
What are you trying to do here? Why are you filtering on a single number? Wouldn't it be better to filter on a particular equipment type?

Reply:
@parangat94: I don't see a response here. Would you please let me know why you're using 137 here. Shouldn't it vary based on the minimum day rate for whatever equipment are chosen?

Reply:
Hi @eberlea, sorry I missed this comment. $137/day here is the cost of renting a walk-behind trencher for a 1 day minimum. This $ value is an _average_ of some of the costs I've seen on various retailers' websites like this one (https://www.homedepot.com/tool-truck-rental/Hydraulic-Trencher-18/E712MTH/index.html). This is only relevant to small DW (that is, < 100 kW turbine rating), and is basically the only equipment used in small DW version of the collection module. 

But yes I agree with you. Filtering on a single equipment rather than cost number will be more a robust implementation. Adding it to my TODO list. 

As mentioned in one of the earlier comments, I have some coding work cut out for myself within the DW space of LandBOSSE (for rest of FY20). Improving code readability, testing/validating, documenting, and enhancing the model are some of my planned tasks within this space. 

Were you expecting this change in implementation by today?

Reply:
Thanks! No need to do this today. Would you just add it as an issue?

Reply:
Sure thing. Doing that now.

Reply:
@parangat94 Have you had a to get rid of the hardcoded 137? No rush from my end, just curious because I still see it in there.

Reply:
Hi @akey7 I've put up an issue for it (#135), but I'm not going to be doing any work on small DW code until the integration is complete. Just because I want to keep duplication of effort to a minimum. Until then, the hardcoded ($) 137 value should be fine; as long as validation passes which I think it does. 

I've currently begin work on writing tests, and created an issue (#136) and branched off of develop as well. I should be done in the next 24-48 hrs or so.